### ISSUE TITLE
Use Liberty & maven and gradle in IntelliJ IDEA

### ISSUE THREAD

**Issue by atosak** [OpenLiberty/open-liberty#8978]
Deliver nice and seamless experience for Maven and Gradle users with IntelliJ.  IntelliJ is a popular if not the most popular IDE for Java developers currently.

Have observed that IntelliJ users working with our guides so the experience should be somewhat alright but there may be bugs or wrinkles to iron out.  IDE likes VS Code and IntelliJ seems to keep a Java process alive -- this may be a bug with our Maven tooling that needs to be fixed.

Should test out the current experience with IntelliJ, identify the gaps and then close them.  Use guides as the proof points of the experience.

Consider delivering a guide for using IntelliJ.  Getting started with developing with IntelliJ and the basics of working with IntelliJ with Open Liberty.  Other relevant collaterals (blogpost?) should be considered too.

**Comment by yeekangc** [comment]
This is working today.  Double checking details and will close the issue as soon as ready.  Will file new epics to track tighter integration if necessary and appropriate.

**Comment by kathrynkodama** [comment]
Confirmed that the Gradle process with IntelliJ is working today.  Walked through the gradle-intro guide and everything worked as expected both with the built-in IntelliJ Gradle plugin and IntelliJ terminal. Tested out the liberty-devmode demo project with Gradle and could easily modify Java, config, and webapp files as well as run IT tests with `gradle check`. Finally tried connecting a debugger and that worked very similar to the VS Code debugger.

Overall, no major issues. The only minor issue I came across was that the java language server on IntelliJ will mark MP Health classes implementing `HealthCheck` as not being used, even though they are automatically detected by MP Health.  This may be confusing to users, however, I am not sure if there is a fix for this.
![image](https://user-images.githubusercontent.com/26146482/68771244-a7e2b380-05f5-11ea-861c-b297a4ba6613.png)
